src/Build/BackEnd/Client/MSBuildClient.cs

```diff
@@ -354,11 +354,11 @@ private bool TrySendPacket(Func<INodePacket> packetResolver)
             {
                 packet = packetResolver();
                 WritePacket(_nodeStream, packet);
-                CommunicationsUtilities.Trace($"Command packet of type '{packet.Type}' sent...");
+                CommunicationsUtilities.Trace("Command packet of type '{0}' sent...", packet.Type);
             }
             catch (Exception ex)
             {
-                CommunicationsUtilities.Trace($"Failed to send command packet of type '{packet?.Type.ToString() ?? "Unknown"}' to server: {0}", ex);
+                CommunicationsUtilities.Trace("Failed to send command packet of type '{0}' to server: {1}", ex, packet?.Type.ToString() ?? "Unknown");
```

Comment:
By the format string it looks like parameters shall be reversed. Type shall be 1st and exception 2nd